diff --git a/node_modules/cloudform-types/types/fms/policy.d.ts b/node_modules/cloudform-types/types/fms/policy.d.ts
index 08b6791..761eece 100644
--- a/node_modules/cloudform-types/types/fms/policy.d.ts
+++ b/node_modules/cloudform-types/types/fms/policy.d.ts
@@ -1,4 +1,4 @@
-import { ResourceBase, ResourceTag } from '../resource';
+import { ResourceBase } from '../resource';
 import { Value, List } from '../dataTypes';
 export declare class IEMap {
     ACCOUNT?: List<Value<string>>;
diff --git a/node_modules/cloudform-types/types/fraudDetector/list.d.ts b/node_modules/cloudform-types/types/fraudDetector/list.d.ts
index 45b47e4..c6d828a 100644
--- a/node_modules/cloudform-types/types/fraudDetector/list.d.ts
+++ b/node_modules/cloudform-types/types/fraudDetector/list.d.ts
@@ -1,10 +1,10 @@
 import { ResourceBase, ResourceTag } from '../resource';
-import { Value, List } from '../dataTypes';
+import { Value, List as ListType } from '../dataTypes';
 export interface ListProperties {
     Description?: Value<string>;
     VariableType?: Value<string>;
-    Elements?: List<Value<string>>;
-    Tags?: List<ResourceTag>;
+    Elements?: ListType<Value<string>>;
+    Tags?: ListType<ResourceTag>;
     Name: Value<string>;
 }
 export default class List extends ResourceBase<ListProperties> {
diff --git a/node_modules/cloudform-types/types/kendra/index.namespace.d.ts b/node_modules/cloudform-types/types/kendra/index.namespace.d.ts
new file mode 100644
index 0000000..4ff9437
--- /dev/null
+++ b/node_modules/cloudform-types/types/kendra/index.namespace.d.ts
@@ -0,0 +1,7 @@
+import DataSource from './dataSource';
+import Faq from './faq';
+import Index from './index';
+
+export declare namespace Kendra {
+    export { DataSource, Faq, Index };
+}
diff --git a/node_modules/cloudform-types/types/resourceExplorer2/index.namespace.d.ts b/node_modules/cloudform-types/types/resourceExplorer2/index.namespace.d.ts
new file mode 100644
index 0000000..bb64312
--- /dev/null
+++ b/node_modules/cloudform-types/types/resourceExplorer2/index.namespace.d.ts
@@ -0,0 +1,7 @@
+import DefaultViewAssociation from './defaultViewAssociation';
+import Index from './index';
+import View from './view';
+
+export declare namespace ResourceExplorer2 {
+    export { DefaultViewAssociation, Index, View };
+}
